docs/owo/setup.md

```diff
@@ -34,13 +34,34 @@ Then, declare the dependency inside your `dependencies` block and as well as the
 === "build.gradle (Neoforge)"
     ```groovy 
     dependencies {
-        // Moddev Projects - Neoforge
+        // Vanilla/Moddev Projects - Neoforge
         implementation "io.wispforest:owo-lib-neoforge:${project.owo_version}"
         accessTransformer "io.wispforest:owo-lib-neoforge:${project.owo_version}"
```

Comment:
```suggestion
        accessTransformers "io.wispforest:owo-lib-neoforge:${project.owo_version}"
```

Reply:
@Noaaan Are you sure as the Mod dev README states otherwise: https://github.com/neoforged/ModDevGradle/blob/12c58a2a8f8fbf03552fe1c7a215ffbd2dd6bcc2/README.md?plain=1#L531

Reply:
I was not able to build it without the s. Users in Discord also experienced the same issue